### DISCUSSION TITLE
How do I retrieve Tweets longer than 280 characters?

### DISCUSSION THREAD

**Discussion by JusticeProject** [tweepy/tweepy#2074] [Questions]
Twitter recently allowed verified users to Tweet more than 280 characters, up to 4000. When I use the API and tweepy to retrieve one of these longer Tweets I don't receive the full text - it's cut off around 280 characters. I don't see any notes about this in the Twitter API documentation, maybe I missed it. Are there any additional fields/expansions I need to send when retrieving one of these longer Tweets? Or does the Twitter API v2 not support this yet?

**Comment by Harmon758** [comment]
As far as I'm aware, there is no way to do so with the API right now, i.e. it's not supported.
